config.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"os"
+
+	log "github.com/Sirupsen/logrus"
+	flag "github.com/spf13/pflag"
+	config "github.com/spf13/viper"
+)
+
+var (
+	confFile      string
+	logLevel      string
+	restAddress   string
+	rpcAddress    string
+	localStateDir string
+)
+
+var (
+	defaultConfPaths = []string{
+		"/etc/glusterd",
+		".",
+	}
+	defaultConfName = "glusterdconf"
```

Comment:
It was a tossup between having conf in the name or not. In the end I felt that the name should give an indication that this was the configuration for GD2, so I kept the conf. But I agree that having it just `glusterd.{yaml|toml|json}` is better. I'll change this.